features/data_converter/failure/README.md

```diff
@@ -0,0 +1,11 @@
+# Failure converter
+
+When using the default failure converter `stack_trace` and `message` can be encoded in the `EncodedAttributes` field.
+
+Steps:
+
+- run a workflow that calls an activity that fails with an error and has a `cause` sub error
```

Comment:
Mention that you also check that `Failure.stack_trace  == ""` and `Failure.message == "Encoded failure"`

Reply:
Let's bubble the error up from the workflow instead of creating a chain of errors in the activity.

Reply:
done!